test/DropdownButtonSpec.js

```diff
@@ -19,7 +20,19 @@ describe('<Dropdown />', () => {
     assert.ok(wrapper.find('.dropdown-content').hasClass('more'));
   });
 
-  it('doesn\'t pass down unwatned props', () => {
+  it('passed `constrainWidth` correctly', () => {
+    const options = { constrainWidth: true };
+    const dropdown = spy();
+    stub(global, '$', () => { return { dropdown }; });
```

Comment:
No need for this has dropdown is already stubbed in the helper, have a look at this example: https://github.com/react-materialize/react-materialize/blob/e34af17a58cedf9a8a22b3c9ab6095537a4ffd2f/test/CarouselSpec.js#L33